Segment.xcodeproj/project.pbxproj

```diff
@@ -348,16 +352,17 @@
 		OBJ_11 /* Segment-Tests */ = {
 			isa = PBXGroup;
 			children = (
```

Comment:
I sorted these by kind so they were alphabetical. Because i'm CDO. Which is like OCD but with the letters arranged alphabetically how they should be.